toolboxes/hdg/cases/poisson/red/test_red.json

```diff
@@ -129,9 +132,12 @@
      },
     "PostProcess":
     {
-        "Exports":
+        "poisson":
         {
-            "fields":["potential","flux"]
+            "Exports":
+            {
+                "fields":["potential","flux"]
+            }
         }
```

Comment:
why not add a Checker Section ? we have a hand made solution from the hdg paper @romainhild